# HG changeset patch
# User David Demelier <markand@malikania.fr>
# Date 1578666142 -3600
#      Fri Jan 10 15:22:22 2020 +0100
# Node ID 7fc487a50780cbedcd5ee1ade676571ce0f6d37a
# Parent  84018b5f2edc3b8dc8f9a87550f5163ddd28f49e
make: -lncurses has to be at the end

diff -r 84018b5f2edc -r 7fc487a50780 Makefile
--- a/Makefile	Mon Oct 14 21:03:08 2019 +0200
+++ b/Makefile	Fri Jan 10 15:22:22 2020 +0100
@@ -38,7 +38,7 @@
 	CC="${CC}" CFLAGS="${CFLAGS}" LDFLAGS="${LDFLAGS}" ./sysconfig.sh > $@
 
 nsnake: ${OBJS}
-	${CC} -o $@ ${LDFLAGS} ${OBJS} ${LIBS}
+	${CC} -o $@ ${OBJS} ${LDFLAGS} ${LIBS}
 
 install: nsnake
 	install -Dm2555 -g ${GID} -o ${UID} nsnake ${DESTDIR}${BINDIR}/nsnake
